core/entities/models/JsonModelSchema.php

```diff
@@ -130,14 +140,90 @@ public function getInitialSchemaStructure()
             'title'      => $this->model->get_this_model_name(),
             'type'       => 'object',
             'properties' => array(),
+            'link' => array(
+                'description' => esc_html__(
+                    'Link to event on WordPress site hosting events.',
+                    'event_espresso'
+                ),
+                'type' => 'string',
+                'readonly' => true,
+            ),
+            '_links' => array(
+                'description' => esc_html__(
+                    'Various links for resources related to the entity.',
+                    'event_espresso'
+                ),
+                'type' => 'object',
+                'readonly' => true,
+                'properties' => array(
+                    'self' => array(
+                        'description' => esc_html__(
+                            'Link to this entities resource.',
+                            'event_espresso'
+                        ),
+                        'type' => 'array',
+                        'items' => array(
+                            'type' => 'object',
+                            'properties' => array(
+                                'href' => array(
+                                    'type' => 'string',
+                                ),
+                            ),
+                            'additionalProperties' => false
+                        ),
+                        'readonly' => true
+                    ),
+                    'collection' => array(
+                        'description' => esc_html__(
+                            'Link to this entities collection resource.',
+                            'event_espresso'
+                        ),
+                        'type' => 'array',
+                        'items' => array(
+                            'type' => 'object',
+                            'properties' => array(
+                                'href' => array(
+                                  'type' => 'string'
+                                ),
+                            ),
+                            'additionalProperties' => false
+                        ),
+                        'readonly' => true
+                    ),
+                ),
+                'additionalProperties' => array(
```

Comment:
we could probably define all the other links, seeing how we know the related models. Do you think that would be worth adding? (Or maybe adding later?)